pkg/addons/data/Deployment.apps/default/docker-registry.yaml

```diff
@@ -10,11 +10,26 @@ spec:
   selector:
     matchLabels:
       docker-registry: default
+  strategy:
```

Comment:
remove, this is set by default in apps/v1

Reply:
as discussed, we need this for updates to be `maxUnavailable`=1 as default is 25%